src/Cli.Tests/Snapshots/UpdateEntityTests.TestConversionOfSourceObject_036a859f50ce167c.verified.txt

```diff
@@ -27,18 +27,12 @@
       MyEntity: {
         Source: {
           Object: s001.book,
-          Type: View
+          Type: View,
+          KeyFields: [
```

Comment:
this seems to be a regression. KeyFields was the old way, Fields is the new way to provide primary key. Same for all snapshot files.